spinners.json

```diff
@@ -1181,5 +1181,165 @@
 			"βββββρβ",
 			"ββββββρ"
 		]
+	},
+	"fingerdance": {
+		"interval": 160,
+		"frames": [
+			"☝️",
+			"✌️",
+			"🤘",
+			"🤟",
+			"🖖",
+			"🖐",
+			"✋",
+			"🤚",
+			"👆"
+		]
+	},
+	"fistBump": {
+		"interval": 80,
+		"frames": [
+			"🤜    🤛",
+			"🤜    🤛",
+			"🤜    🤛",
+			" 🤜  🤛",
+			"  🤜🤛",
+			" 🤜✨🤛",
+			"🤜 ✨ 🤛"
+		]
+	},
+	"header": {
```

Comment:
I think the name could be better and more specific.

Reply:
Alright, I'll change it to "soccerHeader".